### ISSUE TITLE
[question] Exposed members on Polka class

### ISSUE THREAD

**Issue by 43081j** [lukeed/polka#209]
As part of some migrations, i've noticed myself needing the following members from `Polka`:

- `Polka#handler` - so i can bind it to `request` myself rather than calling `listen`
- `wares` - i want to create a new `Polka` which has the same middleware stack as another. for now i've literally set `b.wares = a.wares` but maybe we should have a method for this?
- `onError` - i want to set the error handler _after_ instantiating `Polka`

basically, can i rely on these 3 things being part of the public interface? (i.e. you won't remove them without a new major)

and if not, what could we introduce to cover the same use cases in a supported way?
